Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUQueue.h

```diff
@@ -79,8 +78,7 @@ class Queue : public RefCounted<Queue> {
     virtual void writeBuffer(
         const Buffer&,
         Size64 bufferOffset,
-        void* source,
-        size_t byteLength,
+        std::span<uint8_t> source,
```

Comment:
Why is this one not const given it’s named "source"?

Reply:
That's a good question. There are 2 overloads, one takes a const pointer and the other one doesn't. I didn't spend much time trying to figure out why, but I can investigate.

Reply:
@cdumez @darinadler it is because Metal's API signature requires a non-const pointer:
https://developer.apple.com/documentation/metal/mtldevice/1433382-newbufferwithbytesnocopy?language=objc